Source/Scene/ImplicitTileset.js

```diff
@@ -200,13 +200,13 @@ export default function ImplicitTileset(
   this.subtreeLevels = extension.subtreeLevels;
 
   /**
-   * The deepest level of any available tile in the entire tileset.
+   * The number of levels containing available tiles in the tileset.
    *
    * @type {Number}
    * @readonly
    * @private
    */
-  this.maximumLevel = extension.maximumLevel;
+  this.availableLevels = extension.availableLevels;
```

Comment:
We need to do this in a backwards-compatible way. If available levels is present, use that, otherwise do `availableLevels = extension.maximumLevel + 1` (as maximumLevel is an index, levels is a length).